src/transformers/trainer.py

```diff
@@ -1460,13 +1464,14 @@ def train(
                     f" num_steps ({max_steps}) higher than the number of available samples."
                 )
                 self.control.should_training_stop = True
-            if is_torch_disc_available():
+            if is_torch_ltc_available():
                 import torch._lazy.metrics as metrics
```

Comment:
The naming `metrics` here is the same as the `metrics` at: L1536

Reply:
done